### PR TITLE
add: `mesa-kisak-fresh`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

**Comment by Xdavius** [comment]
need version constraint to provides

**Review by vigress8 — APPROVED** [review]

**Comment by Xdavius** [comment]
Still the pinning problem who is blocking meson ...

File: packages/mesa-kisak-fresh/mesa-kisak-fresh.pacscript [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  "glslang-tools"
+)
+
+pacdeps=("libdrm" "meson-git" "directx-headers-git" "rust-bindgen-git" "wayland-protocols-git")
+repology=("project: mesa")
+maintainer=("xdavius <xdavius@github.com>")
+compatible=("debian:bookworm" "ubuntu:jammy")
+
+prepare() {
+
```
Comment (Elsie19) [inline packages/mesa-kisak-fresh/mesa-kisak-fresh.pacscript:165]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]
LGTM. @oklopfer or @vigress8 could one of you review this as well because it's a fairly large PR?

**Review by oklopfer — APPROVED** [review]

**Pull request merged**